### DISCUSSION TITLE
When I change the language to Arabic,  layout is disorganized

### DISCUSSION THREAD

**Discussion by eissa1005** [serenity-is/Serenity#5917] [Help]
When I change the language to Arabic,  layout is disorganized
![lang1](https://user-images.githubusercontent.com/20481378/129073019-bbb2865f-3c4a-4ccf-844e-23a70551b7d4.png)

**Comment by eissa1005** [comment]
![lang2](https://user-images.githubusercontent.com/20481378/129073509-aa9fbcb0-cadd-40dc-b369-8f61e6f802b7.png)

**Comment by basselz** [comment]
fix it with site.rtl.css

**Comment by eissa1005** [comment]
@basselz 
site.rtl.css not support style with mobile
